ivy/functional/backends/jax/general.py

```diff
@@ -382,10 +382,12 @@ def scatter_nd(
         target = target.at[indices_tuple].min(updates)
     elif reduction == "max":
         target = target.at[indices_tuple].max(updates)
+    elif reduction in ["mul", "prod"]:
+        target = target.at[indices_tuple].mul(updates)
     else:
         raise ivy.utils.exceptions.IvyException(
             "reduction is {}, but it must be one of "
-            '"sum", "min", "max" or "replace"'.format(reduction)
+            '"sum", "min", "max", "mul", "prod", or "replace"'.format(reduction)
         )
```

Comment:
The mode for `mul` won't work without these modes being supported in the `scatter_nd` backends for jax, numpy and tensorflow. As the function is partially mixed, there were some inconsistencies since the `put_along_axis` backends for torch and paddle support these modes, while the compositional implementation doesn't. I was thinking we can at least add the `mul` mode for now and `mean` can be implemented in the future. What do you think ?

Reply:
`scatter_nd` needs to be updated across all backends to support `mean` and `mul`. There is no use in updating only `jax` and `numpy`. So, we will exclude `mul` and `mean` from `put_along_axis` for now. There will be no inconsistency as we will not be testing for these anywhere.